ocaml/parsing/jane_syntax.ml

```diff
@@ -200,7 +224,8 @@ module Comprehensions = struct
      attribute removed. *)
   let expand_comprehension_extension_expr expr =
     match find_and_remove_jane_syntax_attribute expr.pexp_attributes with
-    | Some (comprehensions :: names, attributes)
+    | Some
+        ({ erasability = _; components = comprehensions :: names }, attributes)
```

Comment:
I wonder if `find_and_remove...` shouldn't return the erasability -- we probably don't want to branch on it ever.

Reply:
I addressed this by making `Embedded_name.t` abstract. We can't just have `find_and_remove...` not return the erasability because we want to be able to print the erasability when we print the entire node (see below in `expand_comprehension_extension_expr`.)

Reply:
I suggested making it private in the code, but I think that starts to be a 6-of-1 situation